fuzz/fuzz_targets/router_target.rs

```diff
@@ -110,7 +153,13 @@ pub fn do_test(data: &[u8]) {
 	}
 
 	let logger: Arc<Logger> = Arc::new(test_logger::TestLogger{});
-	let chain_monitor = Arc::new(chaininterface::ChainWatchInterfaceUtil::new(Network::Bitcoin, Arc::clone(&logger)));
+	let input = Arc::new(InputData {
+		data: data.to_vec(),
```

Comment:
Since you don't change the existing calls you end up using elements in data twice, potentially reducing coverage of get_chain_utxo even more.